website/docs/cloud-docs/integrations/kubernetes/api-reference.mdx

```diff
@@ -142,74 +167,86 @@ _Appears in:_
 
 
 
-ConsumerWorkspace allows access to the state for specific workspaces within the same organization. Only one of the fields `ID` or `Name` is allowed. At least one of the fields `ID` or `Name` is mandatory. [More information](/terraform/cloud-docs/workspaces/state#remote-state-access-controls)
+ConsumerWorkspace allows access to the state for specific workspaces within the same organization.
+Only one of the fields `ID` or `Name` is allowed.
+At least one of the fields `ID` or `Name` is mandatory.
+More information:
+  - https://developer.hashicorp.com/terraform/cloud-docs/workspaces/state#remote-state-access-controls
 
 _Appears in:_
 - [RemoteStateSharing](#remotestatesharing)
 
 | Field | Description |
 | --- | --- |
-| `id` _string_ | Consumer Workspace ID. Must match pattern: ^ws-[a-zA-Z0-9]+$ |
+| `id` _string_ | Consumer Workspace ID.<br />Must match pattern: `^ws-[a-zA-Z0-9]+$` |
 | `name` _string_ | Consumer Workspace name. |
 
 
 #### CustomPermissions
 
 
 
-Custom permissions let you assign specific, finer-grained permissions to a team than the broader fixed permission sets provide. [More information](/terraform/cloud-docs/users-teams-organizations/permissions#custom-workspace-permissions)
+Custom permissions let you assign specific, finer-grained permissions to a team than the broader fixed permission sets provide.
+More information:
+  - https://developer.hashicorp.com/terraform/cloud-docs/users-teams-organizations/permissions#custom-workspace-permissions
```

Comment:
```suggestion
Custom permissions let you assign specific, finer-grained permissions to a team than the broader fixed permission sets provide.

More information:
  - https://developer.hashicorp.com/terraform/cloud-docs/users-teams-organizations/permissions#custom-workspace-permissions
```

Reply:
I'm not sure I like these full URLs, why are we switching to this?

Reply:
I'll do a quick pass to clean these link up